### PR TITLE
Newbranch

### PR DESCRIPTION
**Pull request by krisma**
Fixed local date in timeline.
Fixed color in detail view.
Fixed border width.

### CODE REVIEW

File: .io-config.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"app_id":"7c252161","api_key":"3e2dc53de89d0586e307db28007ff543062bb392f1094d75"}
```
Comment (shankari) [inline .io-config.json]: This file is not supposed to be checked in.

Comment (shankari) [inline .io-config.json]: Please remove it from the PR. You can also add it to `.gitignore`

File: bower.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "angular-nvd3": "^1.0.7",
     "string.startsWith": "*",
-    "angularLocalStorage": "^0.3.2"
+    "angularLocalStorage": "^0.3.2",
+    "angular-local-storage": "^0.3.0"
```
Comment (shankari) [inline bower.json]: why do we need the second `angular-local-storage` plugin?

File: ionic.project [hunk 3]
```diff
@@ -1,11 +0,0 @@
-{
-  "name": "e-mission-phone",
```
Comment (shankari) [inline ionic.project]: Why delete this file?

File: package.json [hunk 4]
```diff
@@ -1,94 +0,0 @@
-{
-  "name": "emission",
```
Comment (shankari) [inline package.json]: this should definitely not be deleted.

File: www/js/diary/list.js [hunk 5]
```diff
@@ -117,6 +117,12 @@ angular.module('emission.main.diary.list',['ui-leaflet',
           readAndUpdateForDay(moment(val));
         }
     }
+    $scope.localTimeString = function(dt) {
+      var hr = ((dt.hour > 12))? dt.hour - 12 : dt.hour;
+      var post = ((dt.hour >= 12))? " pm" : " am";
+      var min = (dt.minute.toString().length == 1)? "0" + dt.minute.toString() : dt.minute.toString();
```
Comment (shankari) [inline www/js/diary/list.js]: I wonder if it is easier to continue using the timestamp but with the timezone from the `local_dt`. Care to share your rationale behind this choice?

**Comment by shankari** [comment]
Changes seem fairly straightforward. Basically, fix the files included in the PR and then I can merge.

**Comment by krisma** [comment]
fixed

File: www/js/diary/list.js (hunk 5 above)
Comment (krisma) [inline www/js/diary/list.js]: I think hr+min is more straight-forward. I was about to use 24hr syntax for less code, but then thinking about users would notice the change, I parse it to 12hr.

**Pull request merged**